docs-starlight/src/styles/global.css

```diff
@@ -169,6 +169,13 @@
         padding: 10px 10px;
     }
   }
+
+  @media (min-width: 1152px) {
+    .right-sidebar {
+      padding-top: 0 !important;
+      position: relative !important;
+    }
+  }
```

Comment:
Can we use standard tailwind breakpoints instead of a custom media query? Can we put these styles in the component instead of this `global.css` file which is kind of a grab bag of everything right now?

Reply:
No and no, unfortunately. I tried both. What this is doing is overriding a baked-in starlight style for the right sidebar which is part of their structure and has some very strange markup

Reply:
The design was updated and I was able to remove this. Resolving.